tests/integration/Sample.spec.ts

```diff
@@ -0,0 +1,60 @@
+import "expect-puppeteer";
```

Comment:
Most of these tests really don't assert anything, they just perform actions. What are you *testing*?